resource.html

```diff
@@ -0,0 +1,1051 @@
+                                <div tabindex="0">
+                                    <p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna.</p>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="image base content-image">
+                            <div data-cmp-src="/content/core-components-examples/library/adaptive-form/image/jcr:content/root/responsivegrid/demo/component/guideContainer/container/image_demo.img.jpg/1576866942872.jpg" id="image-8a139445b2" data-cmp-adaptiveformcontainer-path="/content/core-components-examples/library/adaptive-form/image/jcr:content/root/responsivegrid/demo/component/guideContainer" data-cmp-is="adaptiveFormImage" class="cmp-image">
+                                <a>
+                                    <img src="./assets/latest-post1.png" class="cmp-image__image" itemprop="contentUrl">
+                                </a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+                <!-- ARTICLE SLIDER -->
+        <div class="container responsivegrid article-slider aem-GridColumn aem-GridColumn--default--12 layout-grid">
+            <div id="container-b0c5fbd9e2" class="cmp-container">
+                <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+                <!-- ARTICLE MAIN TITLE -->
+                <div class="responsivegrid aem-GridColumn title-btn-wrap aem-GridColumn--default--12 layout-grid">
+                    <div id="container-56884b3bb11" class="cmp-container">
+                    <!-- ARTICLE TITLE -->
+                        <div class="title aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div id="title-bcb754f8400" class="cmp-title">
+                                <h5 class="cmp-title__text">Leading Through a Crisis Video Series</h5>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <!-- ARTICLE SLIDER WRAP -->
+                <div class=" responsivegrid article-slider-wrap aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                    <div id="container-9ce1cf14410" class="cmp-container">
+                    <!-- ARTICLE SLIDER ITEM -->
+                    <div class="container responsivegrid post-item aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                        <div id="container-172911e9c5" class="cmp-container">
+                        <div class="teaser aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div class="cmp-teaser">
+                            <div class="cmp-teaser__image post-article__image">
```

Comment:
@Prystal254 The image/content order needs correcting and the anchor needs to be in the right place. And `.post-article__image` can't go here. It must be on the root of the component.